fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1367,6 +1402,17 @@ def _post_backward_hook(self, param: Parameter, *unused: Any) -> None:
         if param.grad is None:
             return
 
+        if hasattr(param, "_linked_param"):
```

Comment:
where do we set this property?

Reply:
This is set in the kernel, which is the next PR